Unreal/SystemLocalized/det/IpServer.det

```diff
@@ -1,6 +1,6 @@
 ﻿[Public]
 Object=(Name=IpServer.UdpServerUplink,Class=Class,MetaClass=Engine.Actor)
-Preferences=(Caption="Master Server Uplink",Parent="Networking",Class=IpServer.UdpServerUplink,Immediate=True)
+Preferences=(Caption="Master Server Uplink",Parent="Networking",LangCaption="Mit Hauptserver verbinden",Class=IpServer.UdpServerUplink,Immediate=True)
```

Comment:
```suggestion
Preferences=(Caption="Master Server Uplink",Parent="Networking",LangCaption="Masterserver-Uplink",Class=IpServer.UdpServerUplink,Immediate=True)
```

Reply:
Which one should I keep? "Masterserver-Verbindung" or "Masterserver-Uplink"

Reply:
Done then.